service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/CaseInitiationController.java

```diff
@@ -49,10 +48,13 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmitEvent(
 
     @PostMapping("/submitted")
     public void handleSubmittedEvent(
-        @RequestHeader(value = "authorization") String authorization,
-        @RequestHeader(value = "user-id") String userId,
         @RequestBody CallbackRequest callbackRequest) {
 
-        applicationEventPublisher.publishEvent(new InitiatedCaseEvent(callbackRequest, authorization, userId));
+        CaseDetails caseDetails = callbackRequest.getCaseDetails();
```

Comment:
line 51 can go on 50. Then line 53 up to line 51 👍

Reply:
done :)